src/application/TaskDataComparator.java

```diff
@@ -0,0 +1,73 @@
+package application;
```

Comment:
I like this comparator
but can u add in some comments such as "time task = 0, deadline = 1, floating = 2" so that i can expect what i can get from the comparator by a glance? not by scanninf through the entire code?